content/courses/native-onchain-development/paging-ordering-filtering-data-frontend.md

```diff
@@ -147,21 +180,32 @@ For example, you could search through a list of contacts by including a `memcmp`
 filter:
 
 ```tsx
+const DATA_OFFSET = 2; // The first 2 bytes of metadata they store versioning information and are not needed for the search.
```

Comment:
What's the versioning information? Ie, what is being versioned? The account? Something else?

Reply:
fixed ✅